package.json

```diff
@@ -45,6 +45,7 @@
     "build-debug": "webpack",
     "build-min": "cross-env NODE_ENV=production webpack",
     "build-watch": "webpack --watch",
+    "build-minimal-api": "cross-env NODE_ENV=production webpack --config ./minimal-webpack.config.js",
```

Comment:
another small nit here is that we should have one build command instead of 2

Reply:
The webpack do not build to source of files separated, the minimal API have another approach here.

Reply:
I'd agree with @vitormalencar, so I would not suggest this approach here @ktfth, it goes against having a single source of truth inside the project. The idea to have a tiny API to operate the library programmatically is fabulous, but it should live inside the main build. Anyway, thinking about code-splitting in our future architecture, export only a `copy()`/`cut()` method will be awesome in terms of performance.